integration/tests/unowned_cluster/unowned_cluster_test.go

```diff
@@ -3,24 +3,20 @@
 package unowned_clusters
```

Comment:
does this file need to exist anymore?

Reply:
great question! I did delete this at one point, but then I realised this currently tests our support for unowned nodegroups in an unowned cluster. I think in a follow up PR I will move that test into a different suite and delete this :smile: